Tatsi/Views/Assets Grid/AssetsGridViewController.swift

```diff
@@ -417,7 +417,7 @@ extension AssetsGridViewController {
                     return
                 }
                 self.selectedAssets.append(asset)
-                if let maxSelection = self.config?.maxNumberOfSelections, maxSelection == 1 {
+                if let maxSelection = self.config?.maxNumberOfSelections, maxSelection == 1, self.config?.finishImmediatelyWithMaximumOfOne != false {
```

Comment:
There is a difference between `self.config?.finishImmediatelyWithMaximumOfOne != false` and `self.config?.finishImmediatelyWithMaximumOfOne == true` because it can be `nil`. The idea was to keep the original behavior even if it was `nil`.

Reply:
You are right! I forgot about that :)